tools/mo/openvino/tools/mo/ops/TFScatterND.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from openvino.tools.mo.graph.graph import Graph
+from openvino.tools.mo.ops.op import Op
+
+
+class TFScatterND(Op):
+    """
+    TFScatterND operation comes from tf and will be replaced by TFScatterNDDecomposition.
```

Comment:
Add a comment that this is internal operation

Reply:
done